api/src/main/resources/application.yml

```diff
@@ -59,7 +59,7 @@ card:
 
 message-processor:
   # run every 30 seconds. See https://stackoverflow.com/a/26147143 for more examples
-  cron-schedule: "*/30 * * * * *"
+  cron-schedule: "*/2 * * * * *"
```

Comment:
This probably wasn't intended as part of the commit

Reply:
Reverted